### PR TITLE
Push FileService APIs from beta to v1.0

### PR DESCRIPTION
**Pull request by kevklam**

### CODE REVIEW

**Comment by kevklam** [comment]
@daspek

File: api-reference/beta/resources/listinfo.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ Here is a JSON representation of the resource.
 {
   "contentTypesEnabled": false,
   "hidden": false,
-  "template": "documentLibrary | genericList | tasks | survey | links | announcements | contacts | ..."
+  "template": "documentLibrary | genericList | tasks | survey | links | announcements | contacts | ...",
+  "type": "genericList | documentLibrary | discussionBoard | survey | issue | ..."
```
Comment (daspek) [inline api-reference/beta/resources/listinfo.md]: remove for now

File: api-reference/beta/resources/listinfo.md [hunk 2]
```diff
@@ -38,6 +39,7 @@ Here is a JSON representation of the resource.
 | **contentTypesEnabled** | Boolean | If `true`, indicates that content types are enabled for this list.
 | **hidden**              | Boolean | If `true`, indicates that the list is not normally visible in the SharePoint user experience.
 | **template**            | String  | An enumerated value that represents the base list template used in creating the list. Possible values include `documentLibrary`, `genericList`, `task`, `survey`, `announcements`, `contacts`, and more.
+| **type**                | String  | The BaseType of the list
```
Comment (daspek) [inline api-reference/beta/resources/listinfo.md]: remove for now

File: api-reference/v1.0/api/driveitem_list_versions.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+---
+author: rgregg
+ms.author: rgregg
+ms.date: 09/10/2017
+title: List versions of a file
+---
+# Listing versions of a DriveItem (preview)
```
Comment (daspek) [inline api-reference/v1.0/api/driveitem_list_versions.md]: remove preview designation

File: api-reference/v1.0/api/driveitemversion_get.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+---
+author: rgregg
+ms.author: rgregg
+ms.date: 09/10/2017
+title: Get a previous version
+---
+# Get a DriveItemVersion resource (preview)
```
Comment (daspek) [inline api-reference/v1.0/api/driveitemversion_get.md]: remove preview

File: api-reference/v1.0/api/driveitemversion_get_contents.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+---
+author: rgregg
+ms.author: rgregg
+ms.date: 09/10/2017
+title: Download a previous version
+---
+# Download contents of a DriveItemVersion resource (preview)
```
Comment (daspek) [inline api-reference/v1.0/api/driveitemversion_get_contents.md]: remove preview

File: api-reference/v1.0/api/listitem_list_versions.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
+---
+author: rgregg
+ms.author: rgregg
+ms.date: 09/10/2017
+title: Get a previous version of a SharePoint list record
+---
+# Listing versions of a ListItem (preview)
```
Comment (daspek) [inline api-reference/v1.0/api/listitem_list_versions.md]: remove preview

File: api-reference/v1.0/api/listitemversion_get.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
+---
+author: rgregg
+ms.author: rgregg
+ms.date: 09/10/2017
+title: Get a previous version of a list item - SharePoint API
+---
+# Get a ListItemVersion resource (preview)
```
Comment (daspek) [inline api-reference/v1.0/api/listitemversion_get.md]: remove preview

File: api-reference/v1.0/resources/listinfo.md [hunk 8]
```diff
@@ -25,7 +25,8 @@ Here is a JSON representation of the resource.
 {
   "contentTypesEnabled": false,
   "hidden": false,
-  "template": "documentLibrary | genericList | tasks | survey | links | announcements | contacts | ..."
+  "template": "documentLibrary | genericList | tasks | survey | links | announcements | contacts | ...",
+  "type": "genericList | documentLibrary | discussionBoard | survey | issue | ..."
```
Comment (daspek) [inline api-reference/v1.0/resources/listinfo.md]: remove for now

File: api-reference/v1.0/resources/listinfo.md [hunk 9]
```diff
@@ -36,6 +37,7 @@ Here is a JSON representation of the resource.
 | **contentTypesEnabled** | Boolean | If `true`, indicates that content types are enabled for this list.
 | **hidden**              | Boolean | If `true`, indicates that the list is not normally visible in the SharePoint user experience.
 | **template**            | String  | An enumerated value that represents the base list template used in creating the list. Possible values include `documentLibrary`, `genericList`, `task`, `survey`, `announcements`, `contacts`, and more.
+| **type**                | String  | The BaseType of the list
```
Comment (daspek) [inline api-reference/v1.0/resources/listinfo.md]: remove for now

**Pull request closed without merging**